packages/code-du-travail-frontend/package.json

```diff
@@ -18,7 +18,7 @@
     "@cdt/data": "^3.1.1",
     "@cdt/ui": "^3.1.1",
     "@sentry/browser": "^5.6.2",
-    "@socialgouv/kali-data": "^1.0.2",
+    "@socialgouv/kali-data": "^1.1.1",
```

Comment:
bump to 1.2.3 ?

Reply:
on this topic, i hope only the required index.json will be built into bundle...